modules/consentManagementTcf.ts

```diff
@@ -22,11 +24,55 @@ const cmpCallMap = {
   'iab': lookupIabConsent,
 };
 
+export type TCFConsentData = {
+    apiVersion: typeof CMP_VERSION;
+    /**
+     * The consent string.
+     */
+    consentString: string;
+    /**
+     * True if GDPR is in scope.
+     */
+    gdprApplies: boolean;
+    /**
+     * The response from the CMP.
+     */
+    vendorData: Record<string, unknown>;
+    /**
+     * Additional consent string, if provided by the CMP.
+     */
+    addtlConsent?: string;
```

Comment:
```suggestion
    /**
     * Additional consent string, if provided by the CMP.
     * @see https://support.google.com/admanager/answer/9681920?hl=en
     */
    addtlConsent?:`${number}~${string}~${string}`;
```

This cost me too much time of my life